### DISCUSSION TITLE
Create a notification system for uninducted members

### DISCUSSION THREAD

**Discussion by ACascarino** [backstage-technical-services/laravel-site#60] [Discussions]
#### Request

With the decision in the November 2019 committee meeting to add members to the website as soon as they have joined the society, we may now begin to see members sign up to events on the website who have not had a basic H&S induction.
A visual and/or email indicator should be created to notify the committee and the TEM that a member has signed up for an event without having had a basic induction first.

#### Where should this go?

This fits best into the Members section and Events Diary

#### Rationale

This will allow the committee and the TEM to appropriately plan an induction session for these members and reduce the number of last-minute emergency inductions.

#### Urgency

Relatively low priority realistically - having this in place in the inital draft of the SPA would be nice but really it comes into its own during the beginning of the academic year, so if we miss the boat for 2020/21 the deadline moves to the start of the 2021/22 academic year realistically.

**Comment by bnjns** [comment]
I'm not sure how feasible it will be to get v5 out by the beginning of the new year; I'm hopeful seeing as that was the timeframe for v4 and there was significantly more thinking involved, but with work commitments I don't think we can guarantee it.

So I guess a question for @Hypothawits and @ojt29 is how much of a priority this is for next year, as that affects whether we should consider adding this to the MPA?
